packages/evolution-frontend/src/components/pages/auth/AuthPageHOC.tsx

```diff
@@ -72,6 +72,17 @@ const authPageHOC = <P extends AuthPageProps & WithTranslation>(WrappedComponent
             setCurrentAuthMethod(authMethod);
         };
 
+        // Check if there are no auth methods (except directToken)
```

Comment:
You do not really check the directToken part. 

Also `passwordless` is not `true`, but an object.

You should rather filter the auth method for those that are `!== false` and do not `directToken` as key.

Reply:
Done.